source/faq.rst

```diff
@@ -0,0 +1,77 @@
+The quickstart guide is simpler. It has fewer installation steps in favor of predefined default choices. Particularly, in advanced installation guides, you separately apply the Custom Resource Definition and Role-based Access Control configuration files with possible edits in them. At the same time, quickstart guides rely on the all-inclusive bundle configuration.
+
+Also, quickstart guides are related to specific platforms you are going to use (Minikube, Google Kubernetes Engine, etc.), and therefore may include some additional steps needed for these platforms.
+
+Generally, rely on the quickstart guide if you are a beginner user of the specific platform and/or the Percona XtraDB Cluster Operator as a whole.
+
+How HAProxy is better than ProxySQL?
+================================================================================
+
+Percona XtraDB Cluster Operator supports both HAProxy and ProxySQL as a load
+balancer. HAProxy is turned on by default, but both solutions are similar in
+terms of their configuration and operation under the control of the Operator.
+
+Still, they have technical differences. HAProxy is a general and widely used
+high availability, load balancing, and proxying solution for TCP and HTTP-based
+applications. ProxySQL provides similar functionality but is specific to MySQL
+clusters. As an SQL-aware solution, it is able to provide more tight
+internal integration with MySQL instances.
+
+Both projects do a really good job with Percona XtraDB Cluster Operator. The
+load balancer choice should depend mostly on your personal preferences,
+experience with one or another project and its usage in your current
+infrastructure, and (possibly) specific needs of the application.
+
+Why do we need to follow "the Kubernetes way" when Kubernetes was never intended to run databases?
+=====================================================================================================
+
+As it is well known, the Kubernetes approach is targeted at stateless
+applications but provides ways to store state (in Persistent Volumes, etc.) if
+the application needs it. Generally, a stateless mode of operation is supposed
+to provide better safety and sustainability. Moreover, it makes the
+already-deployed components interchangeable.
+
+In the case of state-centric applications (like databases), Kubernetes
+allows to *separate* those parts of the application which *can* be treated as
+stateless (the database engine, etc.) from parts that can not (actual data).
+Depending on the usage scenario, this separation and the appropriate level of
+complexity can bring additional load, like in the case of large databases with
+massive replication, etc. But there are substantial benefits in automatic
+scalability and manageability. Please see `this blog post <https://www.percona.com/blog/2020/10/08/the-criticality-of-a-kubernetes-operator-for-databases/>`_ for more details.
```

Comment:
Only the right architecture of the state-centric application (like databases) can allow avoiding the crash, data loss, or data inconsistency during hardware fail.
It is the main goal of the database operator to solve or simplify these aspects. In a non-Kubernetes environment, issues are the same. Unfortunately, we, Engineers, tend to think that crash, data loss, or data inconsistency issues are not important or not exists in the non-Kubernetes world.
Percona Kubernetes Operator for Percona XtraDB Cluster uses synchronous replication which guarantees consistency on any kind of fail of one cluster member. It has automatic recovery if one member fully or partially lost data. The proxy layer automatically switches traffic to alive members. And many other things which are typically are not solved at all in a typical non-Kubernetes setup.